runtime/interpreter/encoding_test.go

```diff
@@ -85,6 +85,11 @@ func testEncodeDecode(t *testing.T, test encodeDecodeTest) {
 	} else {
 		require.NoError(t, err)
 
+		// Make sure the content is built.
+		if composite, ok := decoded.(*CompositeValue); ok {
+			composite.ensureFieldsLoaded()
```

Comment:
Maybe just call the public `Fields()` function instead, which also calls this private function